base/applications/regedit/lang/hu-HU.rc

```diff
@@ -82,8 +82,8 @@ IDR_REGEDIT_MENU MENU
 BEGIN
     POPUP "&File"
     BEGIN
-        MENUITEM "&Import Registry File...", ID_REGISTRY_IMPORTREGISTRYFILE
-        MENUITEM "&Export Registry File...", ID_REGISTRY_EXPORTREGISTRYFILE
+        MENUITEM "&Import...", ID_REGISTRY_IMPORTREGISTRYFILE
+        MENUITEM "&Export...", ID_REGISTRY_EXPORTREGISTRYFILE
```

Comment:
That is correct. I also noticed that when touching it, but for most of the pther langs I do feel unsure about how to adapt their grammar. I do agree, that it may make sense to shorten most of them, but that's more something for the individual translators. I unfortunately do not speak 30 langauges fluently. I will keep your comment uncollapsed, because I do think it is helpful and inspirational for others.